Code/RDGeneral/utils.h

```diff
@@ -15,7 +15,9 @@
 #include "types.h"
 #include <RDGeneral/Invariant.h>
 #include <RDGeneral/BoostStartInclude.h>
+#define BOOST_ALLOW_DEPRECATED_HEADERS
```

Comment:
Why not put this in BoostStartInclude/BoostEndInclude?  This is where we hide most of the other warnings so they will be used in all of the files that import boost headers.

Reply:
aha! because I wasn't thinking of that. Updating.

Reply:
actually.... Do we want that?

I would think that we'd want to know if we included any boost headers that were deprecated. The current location hides a single known issue. But if, say, in the future `boost::graph::adjacency_list` were deprecated (to pick a crazy example), I would think we'd want to know as soon as possible.